app/controllers/application_controller.rb

```diff
@@ -29,7 +29,12 @@ def categories
 
   def require_log_in
     unless current_user
-      redirect_to new_session_path, alert: "Please log in first." 
+      redirect_to login_path, alert: "Please log in first." 
     end
   end
+
+  rescue_from CanCan::AccessDenied do |exception|
```

Comment:
Unused block argument - `exception`. You can omit the argument if you don't care about it.